tests/integration/cloud/rest_contracts/conftest.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import annotations
```

Comment:
Can we put this file under type checking in our `pyproject.toml`?

Reply:
Is it not by default?

Reply:
@Kilo59

Reply:
No it isn't. The tests we currently type-check are FDS tests and tests for the `/experimental branch`
I agree it should be added.

Reply:
✅